codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/config/ServiceConfigGenerator.kt

```diff
@@ -380,8 +382,9 @@ class ServiceConfigGenerator(
             if (runtimeMode.defaultToOrchestrator) {
                 rustTemplate(
                     """
-                    config: #{CloneableLayer},
-                    runtime_components: #{RuntimeComponentsBuilder},
+                    pub(crate) config: #{CloneableLayer},
+                    pub(crate) runtime_components: #{RuntimeComponentsBuilder},
+                    pub(crate) runtime_plugins: #{Vec}<#{SharedRuntimePlugin}>,
```

Comment:
are these custom/extra runtime plugins or the base ones?

Reply:
These are custom/extra.